Source/WebKit/Scripts/webkit/messages.py

```diff
@@ -848,6 +848,7 @@ def headers_for_type(type):
         'WebCore::CompositeMode': ['<WebCore/GraphicsTypes.h>'],
         'WebCore::CompositeOperator': ['<WebCore/GraphicsTypes.h>'],
         'WebCore::Cookie': ['<WebCore/Cookie.h>'],
+        'WebCore::CookieChangeSubscription': ['<WebCore/CookieStoreManager.h>'],
```

Comment:
May not be needed anymore once you move CookieChangeSubscription to its own CookieChangeSubscription.h header.